src/OrchardCore.Modules/OrchardCore.Users/ViewModels/EditUserViewModel.cs

```diff
@@ -1,13 +1,21 @@
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Localization;
-using MimeKit;
+using OrchardCore.Email;
 
 namespace OrchardCore.Users.ViewModels
 {
     public class EditUserViewModel : IValidatableObject
     {
+        private readonly IEmailAddressValidator _emailAddressValidator;
+
+        public EditUserViewModel(IEmailAddressValidator emailAddressValidator)
```

Comment:
Don't inject services in these models, instead resolve it in the Validate method like it's done for the localizer